roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -81,7 +81,7 @@ mon host = {% if nb_mon > 0 %}
         [{{ hostvars[host]['monitor_address'] }}]
       {%- endif %}
     {%- else -%}
-      {% set interface = 'ansible_' + hostvars[host]['monitor_interface'] %}
+      {% set interface = 'ansible_' + (hostvars[host]['monitor_interface'] | replace('-', '_' ) %}
```

Comment:
You are missing a `)` here, you need to close it.

Reply:
Ah, sorry about that. I'll fix the patch.